libr/bin/p/bin_elf.inc.c

```diff
@@ -195,15 +198,14 @@ static void process_constructors(RBinFile *bf, RList *ret, int bits) {
 		if (type == -1) {
 			continue;
 		}
-		if (sec->size > 0xffffff) {
```

Comment:
this is done to avoid huge allocations. which is an OOM crash in the fuzzing loop

Reply:
I figured, but I only moved it up to save some work if a section is too large